numpy/add_newdocs.py

```diff
@@ -5843,6 +5843,14 @@ def luf(lamdaexpr, *args, **kwargs):
         the result will broadcast correctly against the original `arr`.
 
         .. versionadded:: 1.7.0
+    initializer : scalar, optional
+        The value with which to start the reduction.
+        Defaults to the `~numpy.ufunc.identity` of the ufunc.
+        If ``None`` is given, then the first element of the reduction is used,
```

Comment:
This text is confusing for the reader: we state that the default is the identity, but in the top line it notes `initializer=None` and then here it says that for that case the first element is used. Maybe all it needs is "if explicitly given as ``None``"

Reply:
"if explicitly given as `None`" is not accurate, since it also implies when `identity is None` and the argument is ommited.

Perhaps _"When `None`, either explicitly or when omitted for a ufunc with no identity, ..."_

Reply:
I have a feeling that the complications in the docs are getting out of hand, and may make them hard to parse. Perhaps keep as-is, since we already mention that it defaults to the identity? And change the signature to say `ufunc.identity` instead of `None`?

Reply:
@hameerabbasi - I was thinking the same thing: the problem is really the signature. It makes sense to change it to `ufunc.identity` to me.

Reply:
Done!